Octokit.Tests/Clients/AssigneesClientTests.cs

```diff
@@ -112,6 +112,60 @@ public class TheCheckAssigneeMethod
             }
         }
 
+        public class TheAddAssigneesMethod
+        {
+            [Fact]
+            public async Task PostsToCorrectUrl()
+            {
+                var newAssignees = new AssigneesUpdate(new List<string>() { "assignee1", "assignee2" });
+
+                var connection = Substitute.For<IApiConnection>();
+                var client = new AssigneesClient(connection);
+
+                await client.AddAssignees("fake", "repo", 2, newAssignees);
+
+                connection.Received().Post<Issue>(Arg.Is<Uri>(u => u.ToString() == "repos/fake/repo/issues/2/assignees"), Arg.Any<object>(), "application/vnd.github.cerberus-preview");
+            }
+
+            [Fact]
+            public async Task EnsuresNonNullArguments()
```

Comment:
these need to be reworked to test `null` and `""` values for any applicable field... at the moment `null` is not checked for `owner` or `repo`